balancer.go

```diff
@@ -54,6 +55,35 @@ func (rr *RoundRobin) balance(partitions []int) int {
 	return partitions[offset%length]
 }
 
+// ChunkedRoundRobin is a Balancer implementation that equally distributes messages
+// across all available partitions, but puts greater emphasis on batching by a chunk size
+// within a shorter time period than is possible via the regular RoundRobin Balancer.
+type ChunkedRoundRobin struct {
+	chunkSize int
```

Comment:
I think we want `chunkSize` to be public so that users can configure it

Reply:
Yeah, this is where I thought the test showed you could pass in a value to the struct, but I would find this out for real when I tried to actually test this with a kafka-go based service.  I see that Bulrush uses a setter for this, so I'm sure I'm off base with the way I did this.

I had this worker in mind where I want to be able to pass in a chunk size as they pass in RR as a balancer.  
https://github.com/segmentio/identity/blob/2d04b8f5a16d235453c922e1e9d1ff7ca1b2a92b/identity-resolver/worker.go#L290C21-L290C21

Reply:
The test is in the same package as the balancer so it is able to reference private fields. That won't be the case for users of the balancer. We don't use setters elsewhere in this kafka-go package and just use public fields for fields that we want to expose to users so I'd be inclined to stick with that convention